DSL/org.xtext.ace.acedsl/src/org/xtext/ace/acedsl/generator/AceDslGenerator.xtend

```diff
@@ -181,40 +189,140 @@ class AceDslGenerator extends AbstractGenerator {
+    </style>
+
+    <style name="AppTheme.NoActionBar">
+        <item name="windowActionBar">false</item>
+        <item name="windowNoTitle">true</item>
+    </style>
+    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar" />
+    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
+
+    <style name="DefaultLayout">
+        <item name="android:layout_width">match_parent</item>
+        <item name="android:layout_height">match_parent</item>
+        <item name="android:orientation">vertical</item>
+        <item name="android:layout_marginTop">@dimen/activity_vertical_margin</item>
+        <item name="android:layout_marginBottom">@dimen/activity_vertical_margin</item>
+        <item name="android:layout_marginLeft">@dimen/activity_horizontal_margin</item>
+        <item name="android:layout_marginRight">@dimen/activity_horizontal_margin</item>
+    </style>
+
+    <style name="Button">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">45dp</item>
+        <item name="android:background">@color/color_secondary</item>
+        <item name="android:padding">8dp</item>
+        <item name="android:textColor">@color/color_secondary_contrast</item>
+    </style>
+
+    <style name="TextElement">
+        <item name="android:layout_width">wrap_content</item>
+        <item name="android:layout_height">wrap_content</item>
+        <item name="android:includeFontPadding">false</item>
+        <item name="android:typeface">�estilo.fonte.toString.toLowerCase�</item>
+    </style>
+
+    <style name="TextElement.Header">
+        <item name="android:layout_marginTop">20dp</item>
+        <item name="android:textColor">@color/color_black</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="TextElement.Header.H1">
+        <item name="android:textSize">@dimen/font_size_extra_large</item>
+    </style>
+
+    <style name="TextElement.Header.H2">
+        <item name="android:textSize">@dimen/font_size_large</item>
+    </style>
+
+    <style name="TextElement.Body">
+        <item name="android:layout_marginTop">4dp</item>
+        <item name="android:textSize">@dimen/font_size_small</item>
+        <item name="android:textColor">@color/color_black</item>
+        <item name="android:textStyle">bold</item>
+    </style>
+
+    <style name="TextElement.Paragraph">
+        <item name="android:layout_marginTop">@dimen/activity_vertical_margin</item>
+        <item name="android:layout_marginBottom">@dimen/activity_vertical_margin</item>
+        <item name="android:textSize">@dimen/font_size_medium</item>
+        <item name="android:textColor">@color/color_black</item>
+        <item name="android:lineSpacingMultiplier">1.0</item>
+    </style>
+
+</resources>
+	'''
+
+	def generateAppConstantsTemplate (Aplicativo app) '''
+	package com.example.tcc.tccemptyapp.constants;
+	
+	public class Constants {
```

Comment:
Me avisa quando for modificar esse arquivo, ele é tranquilo de mexer, tem pouca variação